content/src/content/jcr_root/apps/core/wcm/components/experiencefragment/v1/experiencefragment/experiencefragment.html

```diff
@@ -20,6 +20,23 @@
      data-sly-set.selector="content.${request.requestPathInfo.selectorString}"
      data-sly-resource="${@path=fragment.localizedFragmentVariationPath, selectors=selector, wcmmode='disabled'}"
      id="${component.id}"
-     class="cmp-experiencefragment cmp-experiencefragment--${fragment.name}">
+     class="cmp-experiencefragment cmp-experiencefragment--${fragment.name} ${properties.enableRemoteOffer ? ' mboxDefault' : ''}">
 </div>
-<sly data-sly-call="${template.placeholder @ isEmpty=!fragment.configured, classAppend='cmp-dd-experiencefragment'}"></sly>
\ No newline at end of file
+<sly data-sly-call="${template.placeholder @ isEmpty=!fragment.configured, classAppend='cmp-dd-experiencefragment'}"></sly>
+
+<script data-sly-test="${properties.enableRemoteOffer && properties.id}" data-sly-set.mboxId="${properties.id}">
+    adobe.target.getOffer({
```

Comment:
I would make sure the global `adobe.target` is available before calling `getOffer`.

Reply:
@vladbailescu added the check to make sure adobe.target is available before it's called.